fenix/app/src/main/res/drawable/ic_etp_artwork.xml

```diff
@@ -1,164 +1,181 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
No benefit to the code necessarily - anecdotally it can be useful for others maintaining code e.g helps to identify there's no hidden layers or duplicated paths